scania-digital-design-system/sdds
# feat(stepper): migrate stepper component

**Describe pull-request**  
Migrate stepper component.

**Solving issue**  
Fixes: [AB#2404](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2404)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> stepper
3. Check that the design is aligned with the one in Figma.

**Screenshots**  
-

**Additional context**  
-
